src/components/DoubleInput/DoubleInput.js

```diff
@@ -19,80 +18,68 @@ type Props = {|
 */
 
 type Props = {|
-  +onChange: (value: string) => void,
+  +onChangeLeft: (value: string) => void,
+  +onChangeRight: ?((value: string) => void),
   +onClose: ?(() => void),
-  +valueFiat: string,
-  +valueAmount: string,
-  +errorMessageAmount: string,
+  +valueLeft: string,
+  +valueRight: string,
+  +placeholderLeft: string,
+  +placeholderRight: string,
+  +errorMessage: string,
```

Comment:
`errorMessage` is always for both inputs?

Reply:
Yes, and I think we should add the infoMessage (yellow message) here. We need when gasPrice value is too small